src/pm-rpc/__tests__/rpcSpec.js

```diff
@@ -23,7 +23,7 @@ describe('rpc', () => {
     it('should set the API in the appsRegistrar and wait for requests if there is no app with the ID', () => {
       rpc.set(fakeId, fakeAPI)
       expect(appsRegistrar.registerApp).toHaveBeenCalledWith(fakeId, fakeAPI, undefined)
-      expect(messageHandler.addSingleHandler).toHaveBeenCalledWith(jasmine.any(Function), [])
+      expect(messageHandler.addSingleHandler).toHaveBeenCalledWith(jasmine.any(Function), undefined)
```

Comment:
No

Reply:
What no? can you please elabrate

Reply:
Knowingly passing an `undefined` in place of an array is a time bomb